yang-semver/draft-ietf-netmod-yang-semver.txt

```diff
@@ -122,28 +122,58 @@ Internet-Draft                 YANG Semver                  October 2023
    submodule has non-backwards-compatible (NBC) changes compared to its
    previous revision, and a scheme that uses the revision history as a
    lineage for determining from where a specific revision of a YANG
-   module or submodule is derived.  Additionally, section 3.4 of
-   [I-D.ietf-netmod-yang-module-versioning] defines a revision-label
-   which can be used as an alias to provide additional context or as a
-   meaningful label to refer to a specific revision.
-
-   This document defines a revision-label scheme that uses extended
-   semantic versioning rules [SemVer] for YANG artifacts (i.e., YANG
-   modules, YANG submodules, and YANG packages
-   [I-D.ietf-netmod-yang-packages] ) as well as the revision label
-   definition for using this scheme.  The goal being to add a human
-   readable revision label that provides compatibility information for
-   the YANG artifact without needing to compare or parse its body.  The
-   label and rules defined herein represent the RECOMMENDED revision
-   label scheme for IETF YANG artifacts.
+   module or submodule is derived.
+
+   This document defines a YANG extension that tags a YANG artifact
+   (i.e., YANG modules, YANG submodules, and YANG packages
+   [I-D.ietf-netmod-yang-packages] ) with a version identifier that
+   adheres to extended semantic versioning rules [SemVer].  The goal
+   being to add a human readable revision label that provides
+   compatibility information for the YANG artifact without needing to
+   compare or parse its body.  The version identifier and rules defined
+   herein represent the RECOMMENDED approach to apply versioning to IETF
+   YANG artifacts.
 
    Note that a specific revision of the SemVer 2.0.0 specification is
    referenced here (from June 19, 2020) to provide an immutable version.
    This is because the 2.0.0 version of the specification has changed
    over time without any change to the semantic version itself.  In some
    cases the text has changed in non-backwards-compatible ways.
 
-2.  Terminology and Conventions
+2.  Examples of How Versioning Is Applied To YANG Module Revisions
+
+   The following diagram illustrates how the branched revision history
+   and the YANG Semver version extension statement could be used:
+
+   Example YANG module with branched revision history.
+
+          Module revision date        Version Identifier
+            2019-01-01                 <- 1.0.0
+                |
+            2019-02-01                 <- 2.0.0
+                |      \
+            2019-03-01  \              <- 3.0.0
+                |        \
+                |       2019-04-01     <- 2.1.0
+                |           |
+                |       2019-05-01     <- 2.2.0
```

Comment:
FYI Rob has switched the dates of 2.2.0 and 3.1.0 in module-versioning

Reply:
Yeah, I saw that in your updates.  I'll sync the two.

Reply:
Should be in sync now.